src/server/markup.js

```diff
@@ -1,16 +1,20 @@
-export default (reactString, initialData, webserver) => {
+export default (reactString, initialData, webserver, config) => {
+  const stylesheet = process.env.NODE_ENV === 'production'
+    ? `<link rel='stylesheet' href='${webserver}/dist/styles.css'>`
+    : '';
```

Comment:
I think you're right, I had this mind-marked as suboptimal but forgot to revisit. But we might use `react-helmet` later perhaps, which I will tackle with the font-loading story.